.github/workflows/publish-to-winget.yaml

```diff
@@ -61,9 +61,10 @@ jobs:
     steps:
       - name: Create winget PR
         run: |
-          iwr https://aka.ms/wingetcreate/latest -OutFile wingetcreate.exe
-          .\wingetcreate.exe update RedHat.Podman-Desktop -u $Env:URL -v $Env:VERSION -t $Env:TOKEN --submit
+          Invoke-WebRequest https://aka.ms/wingetcreate/latest -OutFile wingetcreate.exe
+          .\wingetcreate.exe update RedHat.Podman-Desktop -u $Env:URL_X64 $Env:URL_ARM64 -v $Env:VERSION -t $Env:TOKEN --submit
```

Comment:
aren't we missing -u in from of the second URL ?

Reply:
AFAIK no, it's a list of URLs
https://github.com/microsoft/winget-create/blob/main/doc/update.md#how-does-winget-create-know-which-installers-to-match-when-executing-an-update